.github/CONTRIBUTING.md

```diff
@@ -1,42 +1,251 @@
+
+```
+[NEW] Allow search permissions and settings by name instead of only ID
+```
+
+Even it's being something new in the code the users already expect the filter to filter by what they see (translations), a better one would be:
+
+```
+[FIX] Permissions' search doesn't filter base on presented translation, only on internal ids
+```
+
+## Choosing the right PR tag
+
+You can use several tags do describe your PR, i.e.: `[FIX]`, `[NEW]`, etc. You can use the descriptions below to better understand the meaning of each one, and decide which one you should use:
+
+### `[NEW]`
+
+#### When
+- When adding a new feature that is important to the end user
+
+#### How
+
+Do not start repeating the section (`Add ...` or `New ...`)
+Always describe what's being fixed, improved or added and not *how* it was fixed, improved or added.
+
+Exemple of **bad** PR titles:
+
+```
+[NEW] Add ability to set tags in the Omnichannel room closing dialog
+[NEW] Adds ability for Rocket.Chat Apps to create discussions
+[NEW] Add MMS support to Voxtelesys
+[NEW] Add Color variable to left sidebar
+```
+
+Exemple of **good** PR titles:
+
+```
+[NEW] Ability to set tags in the Omnichannel room closing dialog
+[NEW] Ability for Rocket.Chat Apps to create discussions
+[NEW] MMS support to Voxtelesys
+[NEW] Color variable to left sidebar
+```
+
+### `[FIX]`
+
+#### When
+- When fixing something not working or behaving wrong from the end user perspective
+
+#### How
+
+Always describe what's being fixed and not *how* it was fixed.
+
+Exemple of a **bad** PR title:
+
+```
+[FIX] Add Content-Type for public files with JWT
+```
+
+Exemple of a **good** PR title:
+
+```
+[FIX] Missing Content-Type header for public files with JWT
+```
+
+### `[IMPROVE]`
+
+#### When
+- When a change enhances a not buggy behavior. When in doubt if it's a Improve or Fix prefer to use as fix.
+
+#### How
+Always describe what's being improved and not *how* it was improved.
+
+Exemple of **good** PR title:
+
+```
+[IMPROVE] Displays Nothing found on admin sidebar when search returns nothing
+```
+
+### `[BREAK]`
+
+#### When
+- When the changes affect a working feature
+
+##### Back-End
+- When the API contract (data structure and endpoints) are limited, expanded as required or removed
+- When the business logic (permissions and roles) are limited, expanded (without migration) or removed
+
+##### Front-End
+- When the change limits (format, size, etc) or removes the ability of read or change the data (when the limitation was not caused by the back-end)
+
+### Second tag e.g. `[NEW][ENTERPRISE]`
+
+Use a second tag to group entries on the change log, we currently use it only for the Enterprise items but we are going to expand it's usage soon, please do not use it until we create a patter for it.
+
+### Minor Changes
+
+For those PRs that aren't important for the end user, we are working on a better pattern, but by now keep the same tags, but use them without the brackets and in camel case:
```

Comment:
... but *for now* please use the same tags;  use them without the brackets and in camel case.